R/ltcad.R

```diff
@@ -0,0 +1,133 @@
+#' @method epidist_prepare epidist_ltcad
+#' @family ltcad
+#' @export
+epidist_prepare.epidist_ltcad <- function(data) {
```

Comment:
Is the action to move these to as_epdisit_ltcad etc for a new issue? If so we need to make one

Reply:
Added issue for this https://github.com/epinowcast/epidist/issues/73